examples/p4est_3d_dgsem/elixir_euler_free_stream_boundaries.jl

```diff
@@ -0,0 +1,60 @@
+
+using Downloads: download
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations3D(1.4)
+
+initial_condition = initial_condition_constant
+
+polydeg = 3
+solver = DGSEM(polydeg = polydeg, surface_flux = flux_lax_friedrichs)
+
+###############################################################################
+# Get the uncurved mesh from a file (downloads the file if not available locally)
+
+default_mesh_file = joinpath(@__DIR__, "cube_boundaries.inp")
+isfile(default_mesh_file) ||
+    download("https://gist.github.com/DanielDoehring/eefe73ae5d113bc3944a518b6e88e663/raw/359a58a808790f3c3efc050273270eb1cc8ee353/cube_boundaries.inp",
+             default_mesh_file)
+mesh_file = default_mesh_file
+
+boundary_symbols = [:PhysicalSurface1]
+
+# Map the unstructured mesh with the mapping above
+mesh = P4estMesh{3}(mesh_file, polydeg = polydeg, boundary_symbols = boundary_symbols)
+
+boundary_conditions = Dict(:PhysicalSurface1 => BoundaryConditionDirichlet(initial_condition))
```

Comment:
Do I understand it correctly that this mesh still has only one boundary? Shouldn't this elixir test that multiple different boundaries work as well?
I'm assuming the only point of this elixir is to test that your code in this PR works in 3D as well?

Reply:
Yes, I was a bit lazy here. I will change this that we have at least two different boundaries when renaming the mesh files.